object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -27,6 +27,8 @@
 
   public static final String CHECK_MUST_CALL = "checkMustCall";
 
+  public static final String Disable_Unconnected_Socket = "disableUnConnectedSock";
```

Comment:
Rename to use all caps:

```suggestion
  public static final String DISABLE_UNCONNECTED_SOCKET = "disableUnConnectedSock";
```

Reply:
You also need to actually not add the `UnconnectedSocketChecker` sub-checker inside `getImmediateSubcheckerClasses()`